internal/config/config.go

```diff
@@ -98,3 +98,43 @@ func PostgresContainerImage(cluster *v1beta1.PostgresCluster) string {
 func PGONamespace() string {
 	return os.Getenv("PGO_NAMESPACE")
 }
+
+// VerifyImageValues checks that all container images required by the
+// spec are defined. If any are undefined, a list is returned in an error.
```

Comment:
How can you tell which images are required by the spec, given what the [CRD](https://access.crunchydata.com/documentation/postgres-operator/latest/references/crd/postgrescluster#postgresclusterspecbackupspgbackrest) says?

Reply:
Since the `RELATED_IMAGES` implementation requires those fields to be optional, it's effectively inferred. The pgBackRest image is always required, either a Postgres or Postgres-GIS image is required and the remainder are determined by your configuration.